overlord/snapshotstate/snapshotstate.go

```diff
@@ -197,6 +202,30 @@ func Restore(st *state.State, setID uint64, snapNames []string, users []string)
 
 	ts = state.NewTaskSet()
 
+	for i, snapID := range snapIDs {
+		if snapID == "" {
+			// snapshotted snap was sideloaded, never mind
+			continue
+		}
+		snapst, ok := all[snapsFound[i]]
+		if !ok {
+			// snap not installed
+			continue
+		}
+		sideInfo := snapst.CurrentSideInfo()
+		if sideInfo == nil {
+			// ... shouldn't happen (but also not installed)
+			continue
+		}
+		if sideInfo.SnapID == "" {
+			// current snap is sideloaded, never mind then
```

Comment:
Same re-wording suggestion as above.

Reply:
I'm going with s/sideloaded/unasserted/, thank you for pointing it out